setup.py

```diff
@@ -7,82 +7,106 @@
 # it under the terms of the Revised BSD License; see LICENSE file for
 # more details.
 
+"""Flask-Menu is a Flask extension that adds support for generating menus."""
+
 import os
-import re
 import sys
 
-from setuptools import setup
+from setuptools import find_packages, setup
 from setuptools.command.test import test as TestCommand
 
+readme = open('README.rst').read()
+history = open('CHANGES.rst').read()
+
+tests_require = [
+    'flask-classy>=0.6.10',
+    'check-manifest>=0.25',
+    'coverage>=4.0',
+    'isort>=4.2.2',
+    'pep257>=0.7.0',
+    'pytest>=2.8.0',
+    'pytest-cache>=1.0',
+    'pytest-cov>=2.1.0',
+    'pytest-pep8>=1.0.6',
+]
+
+extras_require = {
+    'docs': ['sphinx'],
+    'classy': ['flask-classy>=0.6.10'],
```

Comment:
I believe the convention is to keep the list items in separate lines (even single items), with indent and missing trailing commas like here:
https://github.com/jirikuncar/invenio-records/blob/master/setup.py#L72-L79